src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/model/MeterModify.java

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.testing.service.lockkeeper.model;
+
+
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class MeterModify {
+
+    @NonNull
+    String name;
```

Comment:
not clear what `name` is. `switchName` will do better

Reply:
agree, refactored